custom-namespaces.yaml

```diff
@@ -607,6 +607,13 @@ firefox_accounts:
       measures:
         count:
           type: count
+    health_lighthouse:
+      type: table_view
+      tables:
+        - table: mozdata.analysis.wclouser_fxa_health_lighthouse
+      measures:
+        count:
+          type: count
```

Comment:
Do you want to define any other measures/aggregations (e.g. for the `score` or `numVal` columns)?

Reply:
The answer is yes, but I'm not pro enough to know what they look like in yaml yet.  I have to make them by hand in looker first and see what I use. :-/